interface/globals.php

```diff
@@ -4,6 +4,7 @@
 require_once(dirname(__FILE__) . "/../common/compatibility/Checker.php");
 
 use OpenEMR\Checker;
+use OpenEMR\Core\Header;
```

Comment:
Doesn't this basically mean that we will not need to use the OpenEMR\Core\Header in any of the downstream scripts?